components/admin_console/schema_admin_settings.jsx

```diff
@@ -86,6 +86,7 @@ export default class SchemaAdminSettings extends React.PureComponent {
             showConfirmId: '',
             clientWarning: '',
         };
+        this.errorMessage = React.createRef();
```

Comment:
for consistency with the rest of PRs can we rename this to:
```suggestion
        this.errorMessageRef = React.createRef();
```